README.md

```diff
@@ -11,12 +11,12 @@
         <img src="https://img.shields.io/github/actions/workflow/status/neurobagel/annotation_tool/test_e2e.yml?color=8FBC8F&label=e2e test&style=flat-square" alt="e2e test">
     </a>
 	<a href="https://nodejs.org/en//">
-        <img src="https://img.shields.io/badge/node-20.2.0-CD5C5C?style=flat-square" alt="Node">
+        <img src="https://img.shields.io/badge/node-20.9.0-CD5C5C?style=flat-square" alt="Node">
         <a href="https://nuxtjs.org/">
-        <img src="https://img.shields.io/badge/nuxt-2.15.8-F0FFF0?style=flat-square" alt="Nuxt">
+        <img src="https://img.shields.io/badge/nuxt-2.17.2-F0FFF0?style=flat-square" alt="Nuxt">
     </a>
     <a href="https://vuejs.org/">
-        <img src="https://img.shields.io/badge/vue-2.6.14-D8BFD8?style=flat-square" alt="Vue">
+        <img src="https://img.shields.io/badge/vue-2.7.15-D8BFD8?style=flat-square" alt="Vue">
```

Comment:
where do these version bumps come from? Are they manual?

Reply:
Done by dependabot but the badges were lagging behind.